diesel_compile_tests/tests/fail/derive/bad_postgres_type.stderr

```diff
@@ -0,0 +1,80 @@
+error: unexpected end of input, expected parentheses
+ --> $DIR/bad_postgres_type.rs:6:9
+  |
+6 | #[diesel(postgres_type)]
+  |         ^^^^^^^^^^^^^^^
+
+error: expected `oid` and `array_oid` attribute or `name` attribute
+  --> $DIR/bad_postgres_type.rs:10:23
+   |
+10 | #[diesel(postgres_type())]
+   |                       ^^
+
+error: expected parentheses
+  --> $DIR/bad_postgres_type.rs:14:24
+   |
+14 | #[diesel(postgres_type = "foo")]
+   |                        ^
+
+error: unexpected end of input, expected `=`
+  --> $DIR/bad_postgres_type.rs:18:23
+   |
+18 | #[diesel(postgres_type(name))]
+   |                       ^^^^^^
+
+error: expected `=`
+  --> $DIR/bad_postgres_type.rs:22:28
+   |
+22 | #[diesel(postgres_type(name()))]
+   |                            ^^
+
+error: expected string literal
+  --> $DIR/bad_postgres_type.rs:26:31
+   |
+26 | #[diesel(postgres_type(name = Foo))]
+   |                               ^^^
+
+error: unexpected `oid` when `name` is present
+  --> $DIR/bad_postgres_type.rs:30:44
+   |
+30 | #[diesel(postgres_type(name = "foo", oid = 2, array_oid = 3))]
```

Comment:
As a thing for the followup PR: This spans could also be improved using the same strategy as demonstrated for `deserialize_as`/`embed`